Uplift/Views/Onboarding/SignInView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  SignInView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/14/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct SignInView: View {
+
+    // MARK: - Properties
+
+    @EnvironmentObject var mainViewModel: MainView.ViewModel
+    @StateObject private var loginViewModel = LoginViewModel()
+
+    // MARK: - UI
+
+    var body: some View {
+        VStack {
+            signInHeader
+            login
+            Spacer(minLength: 16)
+            skip
+        }
+    }
+
+    private var skip: some View {
+        Button {
+            mainViewModel.displayMainView = true
+        } label: {
+            Text("Skip")
+                .font(Constants.Fonts.bodyNormal)
+                .foregroundColor(Constants.Colors.gray04)
+        }
+    }
+
+    private var login: some View {
+        Button {
+            loginViewModel.googleSignIn { email, name, netId in
+                mainViewModel.email = email
+                mainViewModel.name = name
+                mainViewModel.netID = netId
+                mainViewModel.createUser()
+                mainViewModel.userDidLogin = true
+            }
+        } label: {
+            Text("Log in")
+                .font(Constants.Fonts.h2)
+                .foregroundColor(Constants.Colors.black)
+                .padding(.horizontal, 46)
+                .padding(.vertical, 12)
+                .background(Constants.Colors.yellow)
+                .cornerRadius(38)
+                .upliftShadow(Constants.Shadows.smallLight)
+        }
+    }
+
+    private var cards: some View {
+        VStack(spacing: 12) {
+            goals
+            gymSimple
+            history
+        }
+        .padding(.horizontal, 76)
+    }
+
+    private var goals: some View {
+        HStack {
+            Constants.Images.goal
+            Text("Create fitness goals")
+                .font(Constants.Fonts.f2)
+            Spacer()
+        }
+        .padding(12)
+        .background(.white)
+        .cornerRadius(8)
+        .upliftShadow(Constants.Shadows.smallLight)
+    }
+
+    private var gymSimple: some View {
```

Comment:
Not quite sure what this would do based on the name, could you find a more meaningful name for this